### PR TITLE
chore(monorepo): minor updates in the ci and monorepo implementation

### PR DESCRIPTION
**Pull request by desusai7**
## Description of the change

> minor updates in the ci and monorepo implementation

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Development

- [x] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: melos.yaml [hunk 1]
```diff
@@ -194,7 +194,7 @@ scripts:
       melos run build:integration:firebase --no-select && \
       melos run build:integration:leanplum --no-select && \
       melos bootstrap
-    description: Build the Leanplum integration plugin.
+    description: Build all the integrations.
```
Comment (bardisg) [inline melos.yaml:197]: teh select package in next line seems also unrelated please remove it

Comment (desusai7) [reply melos.yaml:197]: removed it as well

**Review by bardisg — APPROVED** [review]
